docs/useful-robot-commands.md

```diff
@@ -14,6 +15,56 @@
    * [Debugging Uart](#debugging-uart)
    * [Redis](#redis)
 
+# Common Debugging Steps
+```mermaid
+---
+title: Robot Debugging Steps
+---
+flowchart TD
+    ssh(Can you SSH into robot? 
+        `ssh robot@192.168.0.20RobotID` OR
+        `ssh robot@RobotName.local`)
```

Comment:
concrete examples would be nice:

eg: `ssh robot@robert.local` or `ssh robot@192.168.0.203` for a robot called `robert` with robot id `3`